Misc/stable_abi.toml

```
@@ -2507,3 +2507,5 @@
     added = '3.13'
 [function.PyEval_GetFrameLocals]
     added = '3.13'
+[function.Py_TYPE]
+    added = '3.14'
```

Please leave a blank line before the first 3.14 entry :)
```suggestion

[function.Py_TYPE]
    added = '3.14'
```